### ISSUE TITLE
New lines in *.hbs node `innerHTML`

### ISSUE THREAD

**Issue by kcarmonamurphy** [simplabs/emblem-migrator#41]
**Thank you**

Firstly, I'd like to say a huge thank you to the developers of `emblem-migrator`. I work for a medium sized fintech firm and we had over 260 *.emblem files in our repository that we were seeking to migrate. This tool made it quite easy to convert those in batches without disruption to other developers.

**Problem description**

Over the course of our conversions, we noticed several small visual regressions that would crop up after conversion. Notably, some of our layouts were sensitive to whitespace and changes to content in `innerHTML`. We noticed that the converted `hbs` files would sometimes have `\n` characters in the `innerHTML` of nodes. Here's an example:

**Given this content:**
![Screen Shot 2021-11-02 at 9 37 37 AM](https://user-images.githubusercontent.com/1145018/139870176-7d4c5899-6467-48ef-90ba-6a5b459e6e9d.png)

**Before conversion**
![Screen Shot 2021-11-02 at 9 37 30 AM](https://user-images.githubusercontent.com/1145018/139870170-0301f1d9-9c41-4a5f-887f-c4e4777d9042.png)

**After conversion**
![Screen Shot 2021-11-02 at 9 37 42 AM](https://user-images.githubusercontent.com/1145018/139870178-b2cda059-1766-4b12-ab06-77008f3393b3.png)

Not sure if this is an issue with emblem-migrator or the way Handlebars handles whitespaces but I thought it'd raise this here. The workaround for us was to use the whitespace control feature (using `~`) in handlebars (https://handlebarsjs.com/guide/expressions.html#whitespace-control) to remove whitespace before and after tags, although this didn't always solve our issues.

**Comment by Turbo87** [comment]
yeah, that's a known issue caused by the migrator running `prettier` on the templates after the conversion. If we skipped that step the templates would not be readable at all since the emblem compiler was not written for pretty-printing the output.
